src/components/OrgPostCard/components/PostDetailModal.tsx

```
@@ -0,0 +1,273 @@
+          </Button>
+        </div>
+      </Modal.Header>
+
+      <Modal.Body>
+        <div className="row">
+          <div className="col-md-6 mb-3 mb-md-0">
+            {videoAttachment ? (
+              <div
+                style={{ position: 'relative' }}
+                aria-label="Video player with controls"
+              >
```

Biome (2.1.2)

[error] The ARIA attribute 'aria-label' is not supported by this element.

Ensure that ARIA attributes are valid for the role of the element.

(lint/a11y/useAriaPropsSupportedByRole)